content/en/docs/contribution-guidelines/upgrade-graphs.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Upgrade Graphs Diagrams
+linkTitle: Upgrade Graphs
+---
+
+## Standards and definitions
+
+| <div style="align-content: center;">Description</div>   | <div style="align-content: center;width:200px">Diagram Example</div>       | 
+|----------|:-------------:|
+| bundle which is the head of the channel | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::head
+{{</mermaid>}} |
+| bundle which is the installed currently | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::installed
+{{</mermaid>}} |
+| bundle which can be installed | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1)
+{{</mermaid>}} |
+| channel | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph preview
+      ID(v0.0.1) 
+   end
+{{</mermaid>}} |
+| default channel | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph "preview (default)"
+      ID(v0.0.1) --> ID2(v0.0.2)
+   end
+{{</mermaid>}} |
+| replaces action to specify an upgrade path for a project which will replace one version for another. It is defined in the CSV. (eg.`spec.replaces: exampleoperator.v0.0.1`) | {{<mermaid>}}
```

Comment:
IMO users want to understand in the reality what means a replace, skips, skipRange, and OCP label. I mean, if we put ourselves in the user's shoes what are the real motivation for someone check this legend? They would be looking for to understand the diagrams to know how they can apply that in their project. Then, if we do not add the examples info by doing this link we will bring complexity to understand what each item actually represents. See that for they achieve this understanding would need to read a lot of other docs and then, using my self as an example here: when I look replace == cvs.spec.replace sit makes all info much more clear. I can easily get what it actually means. 

However, I agree 100% that it has how to improvements. I will try to address your requests as well.

Reply:
Hi @anik120, 

>  When looking at a graph, do I need to know what replaces means and where I can provide that information?

IMO yes. Why someone would try to understand a graph if is not to know how to apply it to their projects? Also, in POV the info make much more sense and its understanding is much easier when I am able to check that, for example, a replace means cvs.spec.replaces. Otherwise, we cannot pass the same idea instantaneity and users would need to read a lot of docs. 

However, do you think that we need to describe the image ` An arrow from one operator bundle to another`? I think the goal of this column is to let us make a link between the symbol with what that actually means. But we do not describe the image.